tests/util/test_result.rs

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+extern crate rustc_serialize;
+extern crate libstratis;
+
+use libstratis::engine::EngineError;
+use self::rustc_serialize::json::ParserError;
+use std::fmt;
+use std::io::Error;
+
+#[derive(Debug)]
+pub enum TestErrorEnum {
+    _Ok,
+    Error(String),
+    JsonError(String),
+    _NotFound(String),
+}
+
+impl TestErrorEnum {
```

Comment:
See https://github.com/stratis-storage/stratisd/pull/207/commits/b2ab75b804b7c3cdee9ce0c0107a2d44d8883df0 where some improvements are made to the existing error structure. Probably this commit should get off on the right foot by including those improvements.